build-constraints.yaml

```diff
@@ -3584,7 +3581,6 @@ expected-test-failures:
     - shake # Needs ghc on $PATH with some installed haskell packages
     - singletons # Needs ghc on $PATH with som installed haskell packages
     - stack # https://github.com/fpco/stackage/issues/3082
-    - users-persistent # sqlite
```

Comment:
This was here because `users-persistent` needed an sqlite db which we don't provide, does it no longer need that?